llvm/lib/Target/RISCV/RISCVInstrInfo.td

```diff
@@ -1311,6 +1311,22 @@ def : Pat<(FrameAddrRegImm (iPTR GPR:$rs1), simm12:$imm12),
 
 /// HI and ADD_LO address nodes.
 
+let Size = 8, isReMaterializable = 1 in
+def PseudoLIaddr : Pseudo<(outs GPR:$dst), (ins uimm20_lui:$hi, simm12:$lo), []>,
+                     Sched<[WriteIALU]>;
```

Comment:
Don't we need `SchedRead`s here?

Reply:
There are no register operands so they wouldn't be associated with anything.

Reply:
I don't know if we should model the second instruction `ADDI` here.